src/main/java/school/lemon/changerequest/java/introduction/hw1/Task4.java

```diff
@@ -4,6 +4,15 @@
 public class Task4 {
 
     public static double getSumOfGreatest(double a, double b, double c) {
-        return 0;
+        if (a > b && b > c) {
+            return a + b;
+        }
+        if (a > b && b < c) {
+            return a + c;
+        }
+        if (b > a && c < a) {
```

Comment:
In general it's ok, but:
1. First and 3rd returns exactly the same result, it can be combined together. 
2. Can be simpler. You can try to change conditions. There are only 3 combinations, so it should be enough 4 comparison at all. Now you have 6 comparisons.